cni/netconfig.go

```diff
@@ -80,7 +80,9 @@ type NetworkConfig struct {
 }
 
 type WindowsSettings struct {
-	EnableLoopbackDSR bool `json:"enableLoopbackDSR,omitempty"`
+	EnableLoopbackDSR         bool `json:"enableLoopbackDSR,omitempty"`
+	EnableHNSTimeout          bool `json:"enableHNSTimeout,omitempty"`
```

Comment:
we can treat the zero value of the timeout duration as "disable timeouts" - then we don't need two parameters for it

Reply:
that's better, updated it to use that